### DISCUSSION TITLE
消息弹窗控制

### DISCUSSION THREAD

**Discussion by 0kk470** [jynew/jynew#1302] [Ideas]
**功能描述**
用于提供消息提示，以及某些情况下的二次确认操作提示

**UI路径导航**
直接两个按键映射确认和取消，暂时不考虑左右导航
![image](https://user-images.githubusercontent.com/25216715/199403116-1d7e3532-bc2e-4d4a-aa85-a3bbe9dc41a8.png)

**Comment by MephistoSatanDevil** [comment]
不考虑左右导航会不会有点奇怪？

**Reply by 0kk470** [reply]
> 不考虑左右导航会不会有点奇怪？

左右导航的话，相当于就是需要A键来控制确认了，会多一步操作。这个可以取舍下

**Reply by 0kk470** [reply]
Done。 还是用左右导航合适点
